data/json/furniture.json

```diff
@@ -2794,5 +2794,164 @@
                 { "item": "sheet_metal", "count": [0, 2] }
             ]
         }
+    },{
+        "type:" : "furniture",
+        "id" : "f_base_cmd1",
+        "name" : "basic command desk",
+        "symbol" : "#",
+        "bgcolor" : "ltred",
+        "move_cost_mod" : -1,
+        "required_str" : -1,
+        "examine_action" : "base_cmd",
+        "max_volume" : 2000,
+        "flags" : [ "BASE_CMD1", "BASE_CORE", "CMD_SYS", "FLAMMABLE", "ORGANIC" ]
+    },{
+        "type:" : "furniture",
+        "id" : "f_base_cmd2",
+        "name" : "base command desk",
+        "symbol" : "#",
+        "bgcolor" : "ltred",
+        "move_cost_mod" : -1,
+        "required_str" : -1,
+        "examine_action" : "base_cmd",
+        "max_volume" : 500,
+        "flags" : [ "BASE_CMD2", "BASE_CORE", "CMD_SYS", "FLAMMABLE" ]
+    },{
+        "type:" : "furniture",
+        "id" : "f_base_cmd3",
```

Comment:
Merge the first 3 levels - there is no need to keep them separate, especially not before getting the whole thing to work.

Reply:
I have basic levels of function in mind otherwise it's just a difference of power.  Using pen and paper, having a computer and having a fancy console.  Access to a computer creates a large change in functionality.  Upgrading from a laptop to the console was simply going to add access to the aux systems from the command core.  I'll cut the console for now and leave it as pen/paper vs. laptop.